config/src/hinted_off_config.rs

```diff
@@ -21,7 +21,8 @@ impl HintedOffConfig {
     }
 
     fn default_path() -> String {
-        "/var/lib/cnosdb/hh".to_string()
+        let path = std::path::Path::new("/var/lib/cnosdb/hh");
+        path.to_string_lossy().to_string()
```

Comment:
check the  comment https://github.com/cnosdb/cnosdb/issues/1409#issuecomment-1774128452